gravatar/build.gradle.kts

```diff
@@ -119,7 +119,7 @@ openApiGenerate {
             "library" to "jvm-retrofit2",
             "serializationLibrary" to "gson",
             "groupId" to "com.gravatar",
-            "packageName" to "com.gravatar.api",
+            "packageName" to "com.gravatar.restapi",
```

Comment:
What do you think about `restapi` as the name for the new package?

Reply:
I like it 👍